jcl/src/java.base/share/classes/jdk/internal/vm/Continuation.java

```diff
@@ -256,7 +255,11 @@ protected void onContinue() {
 	}
 
 	public boolean isDone() {
-		return finished;
+		return isFinished();
+	}
+
+	public boolean isFinished() {
+		return 0 != (state & CONTINUATION_FINISHED);
```

Comment:
this can be directly placed inside `isDone()`, no reason to have a new public method